setup/nnt_environment.yml

```diff
@@ -1,100 +1,127 @@
-    - numexpr==2.6.9
-    - numpy==1.16.4
-    - opt-einsum==3.1.0
-    - pandas==0.25.0
-    - parso==0.5.1
-    - pillow==6.2.1
-    - pip==19.2.1
-    - protobuf==3.9.1
-    - pydot==1.4.1
-    - pylint==2.3.1
-    - pyparsing==2.4.2
-    - pyserial==3.4
-    - pytz==2019.2
-    - pyyaml==5.1.2
-    - pyzmq==18.1.0
-    - scikit-learn==0.21.3
-    - scipy==1.3.1
-    - six==1.12.0
-    - sklearn==0.0
-    - tables==3.5.2
-    - tensorboard==2.0.0
-    - tensorflow==2.0.0
-    - tensorflow-estimator==2.0.0
+    - decorator==5.0.7
+    - defusedxml==0.7.1
+    - deprecation==2.1.0
+    - entrypoints==0.3
+    - flatbuffers==1.12
+    - future==0.18.2
+    - gast==0.4.0
+    - google-auth==1.29.0
+    - google-auth-oauthlib==0.4.4
+    - google-pasta==0.2.0
+    - grpcio==1.34.1
+    - h5py==3.1.0
+    - idna==2.10
+    - ipykernel==5.5.3
+    - ipython==7.22.0
+    - ipython-genutils==0.2.0
+    - iso8601==0.1.14
+    - jedi==0.18.0
+    - jinja2==2.11.3
+    - joblib==1.0.1
+    - json5==0.9.5
+    - jsonschema==3.2.0
+    - jupyter-client==6.1.12
+    - jupyter-core==4.7.1
+    - jupyter-packaging==0.9.2
+    - jupyter-server==1.6.4
+    - jupyterlab==3.0.14
+    - jupyterlab-pygments==0.1.2
+    - jupyterlab-server==2.4.0
+    - keras-nightly==2.5.0.dev2021032900
+    - keras-preprocessing==1.1.2
+    - kiwisolver==1.3.1
+    - markdown==3.3.4
+    - markupsafe==1.1.1
+    - matplotlib==3.4.1
+    - mistune==0.8.4
+    - nbclassic==0.2.7
+    - nbclient==0.5.3
+    - nbconvert==6.0.7
+    - nbformat==5.1.3
+    - nest-asyncio==1.5.1
+    - notebook==6.3.0
+    - numpy==1.19.5
+    - oauthlib==3.1.0
+    - opt-einsum==3.3.0
+    - packaging==20.9
+    - pandocfilters==1.4.3
+    - parso==0.8.2
+    - pickleshare==0.7.5
+    - pillow==8.2.0
+    - prometheus-client==0.10.1
+    - prompt-toolkit==3.0.18
+    - protobuf==3.15.8
+    - pyasn1==0.4.8
+    - pyasn1-modules==0.2.8
+    - pycparser==2.20
+    - pygments==2.8.1
+    - pyparsing==2.4.7
+    - pyrsistent==0.17.3
+    - pyserial==3.5
+    - python-dateutil==2.8.1
+    - pytz==2021.1
+    - pywin32==225
+    - pywinpty==0.5.7
+    - pyyaml==5.4.1
+    - pyzmq==22.0.3
+    - requests==2.25.1
+    - requests-oauthlib==1.3.0
+    - rsa==4.7.2
+    - scikit-learn==0.24.1
+    - scipy==1.6.2
+    - send2trash==1.5.0
+    - six==1.15.0
+    - sniffio==1.2.0
+    - tensorboard==2.5.0
+    - tensorboard-data-server==0.6.0
+    - tensorboard-plugin-wit==1.8.0
+    - tensorflow==2.5.0rc1
+    - tensorflow-estimator==2.5.0rc0
     - termcolor==1.1.0
-    - typed-ast==1.4.0
-    - werkzeug==0.15.5
-    - wrapt==1.11.2
\ No newline at end of file
+    - terminado==0.9.4
+    - testpath==0.4.4
+    - threadpoolctl==2.1.0
+    - tomlkit==0.7.0
+    - tornado==6.1
+    - traitlets==5.0.5
+    - typing-extensions==3.7.4.3
+    - urllib3==1.26.4
+    - wcwidth==0.2.5
+    - webencodings==0.5.1
+    - werkzeug==1.0.1
+    - wrapt==1.12.1
+prefix: C:\Users\philippgrandeit\anaconda3\envs\nnt
```

Comment:
That's why I usually prefer to use requirements.txt-files instead of environment.yml-files! I'll fix this of course.

Reply:
We could also think about removing the anaconda environment setup.
Seeing as the CI also uses the requirements.txt file, it would give us a bit more freedom and a safety-net without loads of manual testing when changing dependencies.